packages/react/src/use-chat.ts

```diff
@@ -522,18 +525,62 @@ By default, it's set to 0, which will disable the feature.
 
       event?.preventDefault?.();
 
+      let files: MessageFile[] = [];
+
+      if (options.experimental_files) {
```

Comment:
nit: extract `options.experimental_files` into a variable `files`

Reply:
(also below to remove duplication)

Reply:
done!